crates/swc_core/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#![cfg_attr(docsrs, feature(doc_cfg))]
+
+// Quote
+#[cfg(feature = "quote")]
```

Comment:
I think we need `#[cfg(any(docsrs, feature = "quote")]` along with `#[cfg_attr(docsrs, doc(cfg(feature = "quote")))]`